### PR TITLE
fix: cleaning up serializer field inheritance

### PR DESCRIPTION
**Pull request by johnnagro**
## Description

- fixes https://2u-internal.atlassian.net/browse/ENT-6635
- standard, common fields for serialization are defined in the parent serializer but were not consistently being incorporated in the individual implementations
- one side effect was that the `active` field was missing from most of the implementations, causing the UI to show most configurations as disabled
- cleaned up formatting

### CODE REVIEW

File: integrated_channels/api/serializers.py [hunk 1]
```diff
@@ -23,15 +25,18 @@ class Meta:
         fields = (
             'id',
             'display_name',
+            'channel_code',
```
Comment (johnnagro) [inline integrated_channels/api/serializers.py:28]: these additions were in most/all of the implementations, so moving it to the common list

File: integrated_channels/api/v1/blackboard/serializers.py [hunk 2]
```diff
@@ -12,24 +12,18 @@
...
     class Meta:
         model = BlackboardEnterpriseCustomerConfiguration
-        fields = ('id', 'client_id', 'client_secret', 'blackboard_base_url',
-                  'refresh_token', 'transmission_chunk_size', 'uuid', 'enterprise_customer',
-                  'oauth_authorization_url', 'is_valid', 'channel_code', 'active',
-                  'last_sync_attempted_at', 'last_content_sync_attempted_at',
-                  'last_learner_sync_attempted_at', 'last_sync_errored_at',
-                  'last_content_sync_errored_at', 'last_learner_sync_errored_at')
-
+        extra_fields = (
```
Comment (johnnagro) [inline integrated_channels/api/v1/blackboard/serializers.py:18]: field-per-line is easier to read, better diffs

File: integrated_channels/integrated_channel/models.py [hunk 3]
```diff
@@ -282,6 +282,19 @@ def get_last_sync(self, include_only_errors):
         else:
             return None
 
+    @property
+    def is_valid(self):
```
Comment (johnnagro) [inline integrated_channels/integrated_channel/models.py:286]: this should be a commonly defined method that is overridden. opportunities to clean this up but per our discussion in slack, too complicated to clean up now.

**Review by marlonkeating — APPROVED** [review]
Good cleanup!

**Pull request merged**